src/components/Header/index.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import Link from 'next/link';
+import { Link } from '../../../i18n';
```

Comment:
Your path is wrong. Because you are set to default root is 'src' in babel and file i18n is lying outside src. If you want, I would move file i18n to src folder

Reply:
I'm done!